docs/registry/create/_index.md

```diff
@@ -834,6 +1200,17 @@ fn (c *component) someFunction(a int) {
 }
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+Messages sent to `std::cout` in your C++ code are automatically sent to the Viam app over gRPC when a network connection is available.
+
+We recommend that you use a C++ logging library to assist with log message format and creation, such as the Boost trivial logger:
+
+```cpp {class="line-numbers linkable-line-numbers"}
+#include <boost/log/trivial.hpp>
+```
```

Comment:
Per Benji, all `std::cout` messages are silently forwarded up to the Viam app via gRPC. Happy to add any additional context here if you have any!

I hear this is changing in the near/medium term, just file a ticket when this changes and I'll update!